### PR TITLE
Added copyright

### PR DESCRIPTION
**Pull request by watsh-rajneesh**
Added copyright to new/modified files.

### CODE REVIEW

File: verrazzano/tools/databases/mysql/module.xml [hunk 1]
```diff
@@ -19,6 +19,10 @@
   ~ License along with this software; if not, write to the Free
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  ~
+  ~ Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+  ~ Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
```
Comment (mgianatagh) [inline verrazzano/tools/databases/mysql/module.xml:24]: we appear to be adding this twice to the same file

Comment (watsh-rajneesh) [reply verrazzano/tools/databases/mysql/module.xml:24]: Hi Mike, one is mssql and other one is mysql.

File: verrazzano/tools/databases/mariadb/module.xml [hunk 2]
```diff
@@ -19,6 +19,10 @@
   ~ License along with this software; if not, write to the Free
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  ~
+  ~ Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```
Comment (markxnelson) [inline verrazzano/tools/databases/mariadb/module.xml:23]: don't add ours in the middle of theirs like this, let theirs stay as-is, then add ours after it, in the approved format `// ....`

File: verrazzano/tools/databases/mssql/module.xml [hunk 3]
```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+~
+~ Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```
Comment (markxnelson) [inline verrazzano/tools/databases/mssql/module.xml:4]: no tildes

File: verrazzano/tools/databases/mysql/module.xml (hunk 1 above)
Comment (markxnelson) [reply verrazzano/tools/databases/mysql/module.xml:24]: same as other comment

File: verrazzano/tools/databases/oracle/module.xml [hunk 4]
```diff
@@ -19,6 +19,10 @@
   ~ License along with this software; if not, write to the Free
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  ~
+  ~ Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```
Comment (markxnelson) [inline verrazzano/tools/databases/oracle/module.xml:23]: same

File: verrazzano/tools/databases/mariadb/module.xml (hunk 2 above)
Comment (watsh-rajneesh) [reply verrazzano/tools/databases/mariadb/module.xml:23]: Sure will fix it.

**Comment by watsh-rajneesh** [comment]
Fixed all copyright statements in the xml files.

**Pull request merged**